### PR TITLE
Added questions to sql-quiz.ts

### PR DESCRIPTION
**Pull request by sak345**
## Summary of changes

Added 3 questions to the SQL Quiz on WHERE clause, AND logical operator, and UNIQUE constraint. Those 3 questions are added at the end of the sqlQuiz array.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1053,6 +1053,39 @@ const sqlQuiz = [
...
+    Question:
+      "Which SQL clause is used to filter the result set of a SQL query by specifying a search condition?",
+    Answer: "WHERE",
+    Distractor1: "SELECT",
+    Distractor2: "FROM",
+    Distractor3: "GROUP BY",
+    Explanation:
+    "The WHERE clause is an essential part of SQL queries. It allows you to filter the rows in a result set by specifying a search condition that must be met for a row to be included in the result.",
+    Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/#chapter4crudoperationsinsql"
+  },
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1067]: It looks like we already have a similar question in the file.
Please remove this question 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 1 above)
Comment (sak345) [reply src/data/sql-quiz.ts]: Made the specified changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**